### DISCUSSION TITLE
ipxe custom menu does not work on dell_server_c3.medium.x86

### DISCUSSION THREAD

**Discussion by flower01840** [antonym/netboot.xyz#951] [Q&A]
Hi guys,
I meet a problem when I created a c3.medium.x86 (a dell server) on equinix.com and used iPXE script URL: http://netboot.zstack.io/menu.ipxe to boot. But it always stucked on PXE menu and cannot move up and down to select or enter any items.  (PXE menu worked on c3.small.x86, a supermicro server)
I have tested many times and find that after the ipxe scripts chain netboot.xyz-packet.kpxe, the menu works! but the menu is not my custom menu.....
I'm not sure what the problem is, and how to solve it that the menu stucked on dell server.....

**Comment by antonym** [comment]
I know there have been some fixes in iPXE for USB devices if it's having issues with keyboard input, once you load the netboot.xyz-packet.kpxe, you may be getting the latest version of it. The version that initially is loading could be an older version of iPXE.
